addons/web/static/src/legacy/scss/utils.scss

```diff
@@ -468,7 +468,7 @@
 @mixin print-variable($key, $value) {
     @if $value != null {
         $-type: type-of($value);
-        @if $-type == 'string' {
+        @if $-type == 'string' and not str-index($value, 'var(') == 1 {
```

Comment:
Are you sure this is not always false ? :thinking:  The `not` must be evaluated before the `==`, no? ... :disappointed:

Reply:
@anso-odoo

Reply:
@qsm-odoo , suggestion applied, rebased and commit/pr description adapted accordingly.
Thanks